### ISSUE TITLE
LCI inversion

### ISSUE THREAD

**Issue by ishuihan** [gimli-org/gimli#506]
### Problem description

Hello, I want to use pygimli for lateral constraint inversion, but I can't do it. Could you give me an example? thank you!

**Comment by halbmy** [comment]
As there is no such example yet, I would leave this open.

**Comment by ishuihan** [comment]
Well, I thought my question was inappropriate, so you didn't answer it. Thank you for reopening this question.

**Comment by halbmy** [comment]
Not at all. We have been using pyGIMLi for LCI and we should demonstrate that. We first have to bring these examples to the current version. I have it on the list (after some other examples).

**Comment by ishuihan** [comment]
Hi, @halbmy I know you're very busy. The example of LCI inversion may take some time. So I wrote a simple LCI inversion code according to the code in fdem.py, but the inversion result is not ideal. Could you help me check the reason, please? thank you very much.

**Comment by ishuihan** [comment]
[code.zip](https://github.com/gimli-org/gimli/files/11110619/code.zip)
